data/ScaleFactors/Electron_EGamma_SF2D_gsfTracking.json

```diff
@@ -0,0 +1,242 @@
+{
```

Comment:
This file is corrupted. I'm not sure why the Framework accept such file, but the resulting values are kind of random.